mutiny-wasm/src/lib.rs

```diff
@@ -264,6 +264,70 @@ impl MutinyWallet {
         NodeManager::has_node_manager(storage)
     }
 
+    /// Returns the number of remaining seconds until the device lock expires.
+    #[wasm_bindgen]
+    pub async fn get_device_lock_remaining_secs(
+        password: Option<String>,
```

Comment:
Requiring password here? Is that double password requirement then?

Reply:
Is it going to require the user to type in the password twice then? Once for the wallet unlock attempt and then again for the estimation about how long to wait? Why not just return the error with the amount of remaining seconds required?

Reply:
> Is it going to require the user to type in the password twice then? Once for the wallet unlock attempt and then again for the estimation about how long to wait?

No we can cache it in the front end

> Why not just return the error with the amount of remaining seconds required?

because we can't put params in the error for wasm